File: Strings/Python/Remove_common_characters_and_concatenate.py
class Solution:
    #Function to remove common characters and concatenate two strings.
    def concatenatedString(self,s1,s2):
        #code here
        str1 = ""
        for i in s1:
            if i not in s2:
                str1 += i
        for i in s2:
            if i not in s1:
                str1 += i
        if len(str1) == 0:
            return -1
        return str1

File: Solutions/Java/Remove_common_characters_and_concatenate.java
class Solution
{
    //Function to remove common characters and concatenate two strings.
    public static String concatenatedString(String s1,String s2)
    {
        // Your code here
        String res = "";
        for (Character c : s1.toCharArray()) {
            if (!s2.contains(c.toString())) {
                res += c.toString();
            }
        }
        for (Character c : s2.toCharArray()) {
            if (!s1.contains(c.toString())) {
                res += c.toString();
            }
        }
        if (res.length() == 0) {
            return "-1";
        }
        return res;
    }
}
